### PR TITLE
164489882 Setup react router version 5 (React-router-dom)

### PR DESCRIPTION
**Pull request by cop1fab**
## What this PR do?

This PR Setup react router version 4 (React-router-dom)

## How should this be manually tested?
_ This can be manually tested by running the following command npm start and navigating from 
`http://localhost:3000/` to `http://localhost:3000/login`

## What are the relevant pivotal tracker stories?

[#164489882](https://www.pivotaltracker.com/story/show/164489882)

## Screenshots 

image
image

### CODE REVIEW

**Comment by Jaman-dedy** [comment]
I've dropped a comment on the PR, in addition, I think it's better to write the commit in the good format, you can add the body and `Delivers`

**Review by Paccy10 — COMMENTED** [review]
Nice job @cop1fab , but I have a question, Why have you removed the arrow function in `App.js`?

**Comment by cop1fab** [comment]
> I've dropped a comment on the PR, in addition, I think it's better to write the commit in the good format, you can add the body and `Delivers`

Apart from the commit message, what else did you notice?

**Comment by cop1fab** [comment]
> Nice job @cop1fab , but I have a question, Why have you removed the arrow function in `App.js`?

Oh, thanks for this. Let me work on it.

File: src/components/buttons.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (nakiwuge) [inline src/components/buttons.js]: remove this line

File: src/routers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```
Comment (nakiwuge) [inline src/routers.js]: Rename file name to routes.js

File: src/routers.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    BrowserRouter as Router,
+    Route,
+    Switch
+} from 'react-router-dom'
+import App from './App';
+import login from './views/login';
+
+const Routes = () =>(
+    <Router>
+        <h1>Anything that has to appear on all pages</h1>
```
Comment (nakiwuge) [inline src/routers.js]: remove this h1. It is better to have only the routes in this function

File: src/views/login.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+import Buttons from '../components/buttons';
+
+export default class login extends Component {
```
Comment (nakiwuge) [inline src/views/login.js]: It is good practice for class names to begin with a capital letter. Please rename `login` to `Login`

File: src/components/buttons.js (hunk 1 above)
Comment (cop1fab) [reply src/components/buttons.js]: done

File: src/routers.js (hunk 2 above)
Comment (cop1fab) [reply src/routers.js]: done

File: src/routers.js (hunk 3 above)
Comment (cop1fab) [reply src/routers.js]: done

File: src/views/login.js (hunk 4 above)
Comment (cop1fab) [reply src/views/login.js]: Thanks, let me work on this.

**Comment by blaisebakundukize** [comment]
@cop1fab, good work. But notice that you have some comments that @zellymiriam left for changing. Thx

**Review by nakiwuge — CHANGES_REQUESTED** [review]
@copain good job on the implementation, however, for the dummy login page add some input fields and a login button. 
Also, check out the comments I have left on this PR.

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**